src/style.css

```diff
@@ -0,0 +1,56 @@
+:root {
+  --star-size: 16px;
+  --distance-between-stars: 4px;
+  --star-top-offset: 2px;
+  --row-offset: 4px;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  width: calc(5 * var(--star-size) + 4 * var(--distance-between-stars)
+  + var(--row-offset));
+  height: var(--star-size);
+  display: flex;
+  justify-content: space-around;
+}
+
+.stars__star {
+  width: var(--star-size);
+  height: 100%;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: 0 var(--star-top-offset);
+}
+
+.stars--5 > .stars__star {
```

Comment:
Looks a lot better :)
On thing for the future, you could have done something like:
```
.stars--2 > .stars__star:nth-child(-n + 2),
.stars--3 > .stars__star:nth-child(-n + 3),
...
.stars--5 > .stars__star {
  background-image: url("images/star-active.svg");
}
```
You can name all the selectors / selected elements for this rule similarly how you have done this for the starts above